### DISCUSSION TITLE
[SUGGESTION] Shield Generator Visual Improvement

### DISCUSSION THREAD

**Discussion by KiloTheNeko** [tsunamayo/Starship-EVO#4980] [Suggestions]
The shields look good but i think they could look better :P and i will list here my suggestions for improving the way they look

1. add Bloom to them
2. expand the color selection (currently white and black show as blue along with tan being orange)
3.  to show that a vessel has their shields online have a small amount of light refraction and distortion around the ship just enough to make it visible but not intrusive
4. make the on hit indication more fractal like instead of the straight circle it is now

(best image i could find of a 'distorted' line)
![d63c7f_b795042b252b4713956eb163ebc059fd_mv2](https://user-images.githubusercontent.com/65069111/180335475-29ffff8a-adce-4d80-983b-0abf0f143ea7.gif)

(current circle)
![20220722015817_1](https://user-images.githubusercontent.com/65069111/180335525-f2edea85-fbe2-41eb-bbe1-622b56d8b275.jpg)

5. make the shield flash briefly when hit with continued impact keeping it lit

6.
![20220722015844_1](https://user-images.githubusercontent.com/65069111/180335727-02ed940c-e087-485e-bc74-54e63dcf7e99.jpg)
distort the hexagons gradually as they animate like they do now giving the appearance of the shield losing stabilization
with the "re-emerging" effect being much more stable without the distortion 

7. when the shield generator is being hit directly distort it with like a cloud look to it
![ezgif-3-79da943e7a](https://user-images.githubusercontent.com/65069111/180336065-9b11a9d7-e245-4113-9a22-68706de23653.png)
(best image i could find of the look)

of course those are my opinions on how the shields could look better you do not have to agree with me Tsuna :p i still like them as they are

**Comment by tsunamayo** [comment]
@KiloTheNeko Hi, thanks for the screenshot they are great inspiration.
I agree it could be better, for sure I could spend a week on them and make them look better. But unfortunately I am a lone dev, so I need to spend my time efficiently. 
Cheers
